seed/migrations/0132_auto_20201211_1949_squashed_0134_auto_20201211_2053.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.13 on 2020-12-15 19:37
+
+from django.db import migrations, models
+import seed.models.analysis_input_files
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('seed', '0132_auto_20201211_1949'), ('seed', '0133_auto_20201211_2050'), ('seed', '0134_auto_20201211_2053')]
+
+    dependencies = [
+        ('seed', '0131_analysis_analysisinputfile_analysismessage_analysisoutputfile_analysispropertyview'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='analysisinputfile',
+            name='file',
+            field=models.FileField(max_length=500, upload_to=seed.models.analysis_input_files.analysis_input_path),
+        ),
+        migrations.AlterField(
+            model_name='analysis',
+            name='status',
+            field=models.IntegerField(choices=[(8, 'Pending Creation'), (10, 'Creating'), (20, 'Ready'), (30, 'Queued'), (40, 'Running'), (50, 'Failed'), (60, 'Stopped'), (70, 'Completed')]),
+        ),
```

Comment:
I think we can remove this one since we have the 3rd one that includes all the changes.